lib/src/functions/math.dart

```diff
@@ -132,23 +133,15 @@ final _log = _function("log", r"$number, $base: null", (arguments) {
 final _pow = _function("pow", r"$base, $exponent", (arguments) {
   var base = arguments[0].assertNumber("base");
   var exponent = arguments[1].assertNumber("exponent");
-  if (base.hasUnits) {
-    throw SassScriptException("\$base: Expected $base to have no units.");
-  } else if (exponent.hasUnits) {
-    throw SassScriptException(
-        "\$exponent: Expected $exponent to have no units.");
-  } else {
-    return SassNumber(math.pow(base.value, exponent.value));
-  }
+  base.assertNoUnits();
+  exponent.assertNoUnits();
+  return pow(base, exponent);
 });
 
 final _sqrt = _function("sqrt", r"$number", (arguments) {
   var number = arguments[0].assertNumber("number");
-  if (number.hasUnits) {
-    throw SassScriptException("\$number: Expected $number to have no units.");
-  } else {
-    return SassNumber(math.sqrt(number.value));
-  }
+  number.assertNoUnits();
+  return sqrt(number);
 });
 
 ///
```

Comment:
The trig functions below should also call out to the functions in the number library when possible.

Reply:
Nice catch, thanks.